src/physics.js

```diff
@@ -0,0 +1,81 @@
+import AABB from './aabb';
+import QuadTree from './qtree';
+import Victor from 'victor';
+
+export default class Physics {
+    constructor(levelBoundArr, timeStep = 10, gravity = 9.82) {
+        this.physicsBodyArr = new Array();
+        this.timeStepFactor = timeStep/1000;
+        this.gravity = gravity; //units/(s^2)
+        this.quadTree = new QuadTree(0, 0, levelBoundArr[0], levelBoundArr[1]);
+    }
+
+    addBody(physicsBody) {
+        physicsBody.quadID = this.physicsBodyArr.length;
```

Comment:
I see.  It's a unique id that simple-quadree uses for removing items.  Suggest renaming to objectID, uniqueID, or something similar? 

Also I don't know if this is something that could happen, but I'm slightly concerned with the possibility that we might remove something from the PhysicsEngine and then re-add it, causing the new thing to have the same ID as an old thing. 

So....

Suggest adding `this.currentID` to `Physics`, autoincrement whenever `addBody` is called and set the ID to that.  That way even if we remove and re-add something, the new ID will be guaranteed unique from any previous.